docs/user_guide/assessment.rst

```diff
@@ -1,21 +1,109 @@
 Assessment
 ==========
 
```

Comment:
The document begins with two headers: Assessment and Introduction. Perhaps we should hide the Assessment header, or is this a bad idea?

Reply:
I don't think we need it, so I've removed it. I can revert though if needed.